packages/client/src/pages/CheckerPage.tsx

```diff
@@ -111,6 +120,10 @@ const CheckerPage = () => {
     return <LoadingPage />;
   }
 
+  if (!sessionContext.session[slug]) {
```

Comment:
Why not test for topicData instead? Importing sessionContext is not needed then?

Reply:
Because the whole problem is that topicData can be defined and holds the data of another slug. That's why this check is needed.

Reply:
I will add a comment

Reply:
Or when someone manually deletes the Session Storage